R/pkg-arrow.R

```diff
@@ -39,11 +39,8 @@ ArrowSubstraitCompiler <- R6::R6Class(
 
         name <- unname(binary_map[[name]])
       } else {
-        stop(
-          sprintf(
-            "Don't know how to convert call to `%s` to Arrow",
-            name
-          )
+        rlang::abort(
+          paste0('Could not find function "',  name, '"')
```

Comment:
{dplyr} imports {glue} so {substrait-r} could add it for "free". Had a quick look and at the moment there doesn't seem to be a lot of `paste0`ing going on so 🤷  but maybe something to keep in mind for the future?

Reply:
Sure, though I'm pretty happy with `paste()` and `paste0()` here, it's all pretty readable as-is.